### DISCUSSION TITLE
Introduction to Paging (zh-CN)

### DISCUSSION THREAD

**Discussion by giscus** [phil-opp/blog_os#1326] [Post Comments (translated)]
# Introduction to Paging (zh-CN)

本文主要讲解 内存分页 机制，一种我们将会应用到操作系统里的十分常见的内存模型。同时，也会展开说明为何需要进行内存隔离、分段机制 是如何运作的、虚拟内存 是什么，以及内存分页是如何解决内存碎片问题的，同时也会对x86_64的多级页表布局进行探索。

https://os.phil-opp.com/zh-CN/paging-introduction/

**Comment by miaomiao1992** [comment]
very nice !
